workflow/kfdrc_consensus_calling.cwl

```diff
@@ -92,8 +96,7 @@ steps:
       tumor_id: input_tumor_name
       output_basename: output_basename
       reference: indexed_reference_fasta
-      retain_info:
-        valueFrom: ${return "MQ,MQ0,CAL,HotSpotAllele"}
+      retain_info: maf_retain_info
```

Comment:
I am also working on adding an optional `retain_fmt` input, but that should probably be a seperate PR